sites/docs/pages/reference/themes/index.md

```diff
@@ -0,0 +1,524 @@
+---
+sidebar_position: 4
+hide_table_of_contents: false
+title: Themes
+description: Customize the appearance of your Evidence application in light and dark mode using our theming system.
+sidebar_badge: New!
+---
+
+Customize the appearance of your Evidence application in light and dark mode using our theming system.
+
+# Appearance
+
+Evidence supports three appearance modes: `light`, `dark`, and `system`. An unconfigured Evidence application will use `light` mode and not allow switching appearances. Evidence includes a built-in dark mode without any configuration necessary.
+
+When the appearance is `system`, the user's preferred appearance from their operating system is used.
+
+To enable appearance modes in your app, add the following to your `evidence.config.yaml` file:
+
+```yaml
+appearance:
+    default: system 
+    switcher: true
+```
+
+## Options
+
+<PropListing
+    name="appearance.default"
+    description="The default appearance mode."
+    options={['dark', 'light', 'system']}
+    defaultValue="light"
+/>
+<PropListing
+    name="appearance.switcher"
+    description="Enables/disables the appearance switcher in the kebab menu in the top right, allowing users to switch the appearance of your application between light and dark mode."
+    options={['true', 'false']}
+    defaultValue="false"
+/>
+
+# Theme
+
+The theme configuration defines the colors used by your application for everything on the page including the background, text, inputs, and charts.
+
+The theme consists of 3 sections that define colors for different purposes:
+
+- `colorPalettes` configures colors for charts with different data series (e.g. [Bar Charts](/components/bar-chart/#props-colorPalette), [Line Charts](/components/line-chart/#props-colorPalette)).
```

Comment:
```suggestion
- `colorPalettes` configure colors for charts with different data series (e.g. [Bar Charts](/components/bar-chart/#props-colorPalette)).
```

Reply:
This is written like a sentence, "color palettes configures", not like a definition `color palettes: configure`. I'd prefer to leave it as I think its easier to read this way.

And why remove the bar charts link?